noWord/common/DefaultTransformation.py

```diff
@@ -52,18 +54,55 @@ def distinct( input, params, context):
     return outputRes
 
 
+def selectFirst( input, params, context):
+
+    inputRes = context.getResource(context.resources, input)
+
+    outputRes = {}
+
+    for item in inputRes :
+        if params['key'] in item :
+            if item[params['key']] == params['value']:
+                outputRes = item

```

Comment:
I propose this features for the following reasons:
- I hade to execute multiple transformations on the same resource: the yaml file was becoming very long and verbose.
- I hade to invent unnecessary resources names for intermediate resources 
- I would like to create a list of very basic transformation allowing in combination to achieve advanced transformation scenarios. This can also be achieved by directly implementing complex transformations in python.

I'm open for the debate, It is only a proposition. With this proposition, my yaml file have become much more consistent and readable.

Reply:
Good point indeed!

Reply:
(the missing break)

Reply:
DONE

Reply:
> I would like to create a list of very basic transformation allowing in combination to achieve advanced transformation scenarios. This can also be achieved by directly implementing complex transformations in python.

I do agree with this argument. I my opinion, the chained transformation is much better than implementing very specific and complex transformations.